audit/projects/kubernetes-public/buckets/k8s-infra-clusters-terraform/iam.json

```diff
@@ -2,8 +2,7 @@
   "bindings": [
     {
       "members": [
-        "group:k8s-infra-gcp-org-admins@kubernetes.io",
-        "user:spiffxp@google.com"
+        "group:k8s-infra-gcp-org-admins@kubernetes.io"
```

Comment:
Expected, this was the cleanup I did in https://github.com/kubernetes/k8s.io/pull/2001#discussion_r626857955